clang/lib/CIR/CodeGen/CIRGenBuilder.h

```diff
@@ -959,6 +949,19 @@ class CIRGenBuilderTy : public CIRBaseBuilderTy {
     return create<mlir::cir::GetRuntimeMemberOp>(loc, resultTy, objectPtr,
                                                  memberPtr);
   }
+
+  /// create a cir.ptr_stride operation to get access to an array element.
```

Comment:
Comments should in general start with upper case, please apply the change here, next comment, and try to make sure next PRs already address that.